provider/storage_benchmark.py

```diff
@@ -281,7 +281,9 @@ def __init__(self, params, session):
         host_path = os.path.join(data_dir.get_deps_dir(), 'iozone', iozone_pkg)
         self.download_path = os.path.join('/home', iozone_pkg)
         self.iozone_inst = os.path.join('/home', 'iozone_inst')
-        self.arch = 'linux-AMD64' if 'x86_64' in machine() else 'linux-powerpc64'
+        self.arch = 'linux-AMD64' if 'x86_64' in machine() \
```

Comment:
please split to multi-lines. use if else branch instead.

Reply:
@xutian could you explain more ?  what's the defect of the current code since both of them can work.